src/layout/Summary2/CommonSummaryComponents/MultipleValueSummary.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import { ErrorMessage, Label, List, Paragraph } from '@digdir/designsystemet-react';
+import cn from 'classnames';
+
+import { Lang } from 'src/features/language/Lang';
+import { EditButton } from 'src/layout/Summary2/CommonSummaryComponents/EditButton';
+import classes from 'src/layout/Summary2/CommonSummaryComponents/MultipleValueSummary.module.css';
+import type { BaseValidation } from 'src/features/validation';
+import type { LayoutNode } from 'src/utils/layout/LayoutNode';
+
+type MultipleValueSummaryProps = {
+  title: React.ReactNode;
+  errors: BaseValidation[];
+  componentNode: LayoutNode;
+  displayData?: string;
+  showAsList?: boolean;
+};
+
+export const MultipleValueSummary = ({
+  title,
+  errors,
+  componentNode,
+  displayData,
+  showAsList,
+}: MultipleValueSummaryProps) => (
+  <div className={classes.checkboxSummaryItem}>
+    <div className={cn(classes.labelValueWrapper, { [classes.error]: errors.length > 0 })}>
+      <Label weight={'regular'}>{title}</Label>
+      {displayData && showAsList && (
+        <List.Root>
+          <List.Unordered>
+            {displayData?.split(',').map((item) => (
```

Comment:
I'm pretty sure splitting display values by comma will break. Introducing a code list with values such as `Stavanger, Rogaland` would split up into two list items when it was supposed to only be one. I would suggest taking the raw option values (that can be split by commas) and get the option labels for each of them, and then pass that as an array.

Reply:
But wont the data be cleaned and processed and turned into a reliable comma separated list by the time it reaches a summary component? Also, if we do it the way you suggest, how will we know what raw option values are selected when we only get the display data as a string?

Reply:
> But wont the data be cleaned and processed and turned into a reliable comma separated list by the time it reaches a summary component?

"Displaydata"/"displayValue" is a human-readable string, so when you have options like:

```
[
  { label: "Stavanger, Rogaland", value: 1103 },
  { label: "Sandnes, Rogaland", value: 1102 }
]
```

And you check both options in a Checkboxes component, the 'form data' (what is actually stored in the data model) will be `1103,1102`. But the "display data" value is much less strictly defined. It doesn't really have to follow any rules, but I would assume the display data here would be `Stavanger, Rogaland, Sandnes, Rogaland` (but yes, that's also hard for humans to read). I've seen app developers try to cram long strings with commas in them as option _values_, so I can all but guarantee there are apps with commas in option _labels_.

> Also, if we do it the way you suggest, how will we know what raw option values are selected when we only get the display data as a string?

What I'm saying is _do not get the display data as a string_ :wink: That hook is not meant for this use-case, so you need to make something more appropriate if you want to display each selected option as a list - `displayValue`.